### DISCUSSION TITLE
Hardware for OctoPod

### DISCUSSION THREAD

**Discussion by cfgroenewald** [gdombiak/OctoPod#603] [Q&A] [answered]
Hi
What hardware do I need to add to a SKR pro 1.2 to use this please?

**Comment by gdombiak** [comment, marked as answer]
Hi @cfgroenewald,

Your BIGTREETECH SKR PRO V1.2 is a great upgrade for your printer! Once you installed it on your printer and flashed latest (and configured) version of Marlin you are ready to install OctoPrint. You will need to get a Raspberry Pi 3 or newer and [follow this guide](https://octoprint.org/download/). Once you have OcroPrint running you can then use OctoPod from your iPhone or iPad.

Regards,
Gaston

**Comment by cfgroenewald** [comment]
Thanks so much Gastron. No to find a Raspberry Pi somewhere.
